source/04_best_practice/ddp.rst

```diff
@@ -0,0 +1,147 @@
+# Distributed Training with DDP in DI-engine  
+===========================================  
+
+When accelerating the reinforcement learning (RL) training process, **Distributed Data Parallel (DDP)** is an effective approach.  
```

Comment:
**Distributed Data Parallel (DDP)** is an effective approach for accelerating the reinforcement learning (RL) training process.